src/services/qix/rpc.js

```diff
@@ -104,7 +104,7 @@ class RPC {
   onClose(event) {
     this.emit('closed', event);
     this.resolvers.closed.resolveWith(event);
-    this.rejectAllOutstandingResolvers();
+    this.rejectAllOutstandingResolvers(event);
```

Comment:
These pending QIX method calls would likely expect a reject with an error object that follows the QIX protocol, we might want to simulate that here?